tests/foreman/api/test_role.py

```diff
@@ -16,15 +15,13 @@
 )
 from nailgun import entities
 from requests.exceptions import HTTPError
-from robottelo import decorators
+from robottelo.decorators import bz_bug_is_open
 from robottelo.test import APITestCase
 
 
-@ddt.ddt
-class RoleTestCase(APITestCase):
-    """Tests for ``api/v2/roles``."""
-
-    @decorators.data(
+def data_generator():
```

Comment:
Here `data` is plural and `generator` is singular.

This is a single generator returning different types of data.